amethyst_renderer/src/pass/sprite/separate.rs

```diff
@@ -0,0 +1,157 @@
+//! Simple flat forward drawing pass.
+
+use amethyst_assets::AssetStorage;
+use amethyst_core::specs::prelude::{Entities, Join, Read, ReadStorage};
+use amethyst_core::transform::GlobalTransform;
+use gfx_core::state::{Blend, ColorMask};
+use glsl_layout::Uniform;
+
+use super::*;
+use cam::{ActiveCamera, Camera};
+use error::Result;
+use mtl::MaterialTextureSet;
+use pass::util::{draw_sprite, get_camera, setup_textures, SpriteArgs, VertexArgs};
+use pipe::pass::{Pass, PassData};
+use pipe::{DepthMode, Effect, NewEffect};
+use sprite::{SpriteRender, SpriteSheet};
+use tex::Texture;
+use types::{Encoder, Factory};
+use visibility::Visibility;
+
+/// Draw mesh without lighting
```

Comment:
I'm not too familiar with the difference between a normal and a "separate" pass. Would it be possible to add documentation explaining the difference between the 2?

Reply:
> I'm not too familiar with the difference between a normal and a "separate" pass.

you and me both :stuck_out_tongue:. This was me copying `DrawFlat`. I'll have a go

* [ ] docs for separate vs interleaved passes

Reply:
So I figured out what it's for :stuck_out_tongue:, and documented what it is in `amethyst_renderer/src/lib.rs`. I've deleted `DrawSpriteSeparate` because it isn't necessary.